config/abilities/provider_any.rb

```diff
@@ -1,7 +1,9 @@
-# here we define abilities for all users of provider account, for members use
-# provider_member.rb
+# frozen_string_literal: true
+
+# here we define abilities for all users of provider account, for members use provider_member.rb
+# rubocop:disable Metrics/BlockLength
 Ability.define do |user|
```

Comment:
To avoid having to enable the check again, you can disable it for the block:
```suggestion
Ability.define do |user| # rubocop:disable Metrics/BlockLength
```